### PR TITLE
Admins can now apply for a named role for a user

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: pages/profile/index.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ const list = require('./list');
 const read = require('./read');
 const invite = require('./invite');
 const invitations = require('./invitations');
-const role = require('./permission');
+const permissionRouter = require('./permission');
```
Comment (lennym) [inline pages/profile/index.js]: Can you remove the `Router` bit for consistency with the other routers?

File: pages/profile/read/views/index.jsx [hunk 2]
```diff
@@ -85,61 +91,62 @@ const Index = ({
...
+                              roles.map(({ type, id }) =>
+                                <li key={id}>{defineValue(type.toUpperCase())}</li>
+                              )
+                            }
+                          </ul>
+                        }
+                        { isEmpty(roles) &&
+                          <Snippet>responsibilities.noRoles</Snippet>
+                        }
+                        <RoleApply />
```
Comment (lennym) [inline pages/profile/read/views/index.jsx]: This should only be visible if the logged in user has permission to edit roles.

Comment (lennym) [reply pages/profile/read/views/index.jsx]: Actually, I see the component itself handles that. That's probably a better way of doing it 👍

File: pages/profile/roles/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    return req.api(`/establishment/${req.establishmentId}/profile/${req.profileId}/role`, opts)
+      .then(() => res.redirect(req.buildRoute('profile.role.success')))
+      .catch(next);
+  });
+
+  app.use('/success', success({ model: 'role' }));
+
+  return app;
+};
```
Comment (lennym) [inline pages/profile/roles/index.js]: There should be a permissions check somewhere in this router.

Comment (wheelsandcogs) [reply pages/profile/roles/index.js]: As discussed offline, the permissions check is taken care of when this router is mounted in `pages/profile/index.js`.

File: pages/profile/index.js (hunk 1 above)
Comment (lennym) [reply pages/profile/index.js]: Makes sense. Leave it be then.

**Pull request merged**